src/app/account/invitation/[inviteId]/page.tsx

```diff
@@ -27,13 +26,20 @@ export default async function AcceptInvitePage({ params }: { params: Promise<{ i
         .where('pendingUser.id', '=', inviteId)
         .executeTakeFirst()
 
-    if (!invite) {
-        return <AlertNotFound title="Unable to signup" message="unable to find invitation" />
+    const claimedInvite = await db
+        .selectFrom('pendingUser')
+        .where('claimedByUserId', 'is not', null)
+        .where('id', '=', inviteId)
+        .executeTakeFirst()
+
+    // if the invite has been claimed by a user, redirect to the signin page
+    if (claimedInvite) {
+        redirect(`/account/signin`, RedirectType.replace)
```

Comment:
should we show an error here?  We probably don't want to say "invite has been used ", but could just say "Invite not found" like the earlier message